Doc/whatsnew/3.12.rst

```diff
@@ -147,16 +147,42 @@ Modules (see :pep:`594`):
 * :mod:`uu`
 * :mod:`xdrlib`
 
+Other modules:
+
+* :mod:`lib2to3`, and the ``2to3`` program (:gh:`84540`)
+
 APIs:
 
 * :class:`configparser.LegacyInterpolation` (:gh:`90765`)
 * :func:`locale.getdefaultlocale` (:gh:`90817`)
 * :meth:`turtle.RawTurtle.settiltangle` (:gh:`50096`)
 * :func:`unittest.findTestCases` (:gh:`50096`)
-* :func:`unittest.makeSuite` (:gh:`50096`)
 * :func:`unittest.getTestCaseNames` (:gh:`50096`)
+* :func:`unittest.makeSuite` (:gh:`50096`)
+* :meth:`unittest.TestProgram.usageExit` (:gh:`67048`)
 * :class:`webbrowser.MacOSX` (:gh:`86421`)
 
+Pending Removal in Python 3.14
+==============================
+
+The following modules and APIs have been deprecated in earlier Python releases,
+and will be removed in Python 3.14.
+
+APIs:
+
+*  :meth:`asyncio.Future.cancel`'s *msg* parameter (:gh:`90985`)
+*  :meth:`asyncio.Task.cancel`'s *msg* parameter (:gh:`90985`)
+
+Pending Removal in Python 4.0
```

Comment:
Can we deprecate this in a `3.xy` release? gh-57281 is from 2012 and beliefs about Python 4 were different a decade ago.